client/guidestours/index.js

```diff
@@ -183,22 +290,32 @@ export default class GuidesTours extends Component {
 			return null;
 		}
 
-		const { bullseye, dialog } = this.getStepPositions();
-		const stepCoords = posToCss( dialog );
-		const pointerCoords = posToCss( bullseye );
+		let StepComponent = GuidesStep;
+		switch ( this.state.currentStep.type ) {
+			case 'GuidesFirstStep':
```

Comment:
I wonder whether it would make more sense putting these references in the config file. So `this.state.currentStep.component` would be a reference to the actual component.